src/pages/subjects/SubjectListPage.jsx

```diff
@@ -57,23 +60,26 @@ const SubjectListPage = ({ t }) => {
 				{...searchInput}
 			/>
 
-			<StyledDiv as="section">
+			<StyledDiv as="section" {...section}>
 				<SubjectListSectionTitle>{t('subjects.list_page.sections.active_subjects')}</SubjectListSectionTitle>
 				<StyledList {...sectionList}>
 					{subjects?.active && subjects?.active.length > 0 && subjects.active.map((subject) => (<SubjectCard key={subject.id} {...subject} />))}
 					{(!subjects?.active || subjects.active.length === 0) && <BodyCopy>{t('subjects.list_page.no_data')}</BodyCopy>}
 				</StyledList>
 			</StyledDiv>
 
-			<StyledDiv as="section">
+			<StyledDiv as="section" {...section}>
 				<SubjectListSectionTitle>{t('subjects.list_page.sections.my_subjects')}</SubjectListSectionTitle>
 				<StyledList {...sectionList}>
 					{subjects?.mine && subjects?.mine.length > 0 && subjects.mine.map((subject) => (<SubjectCard key={subject.id} {...subject} />))}
 					{(!subjects?.mine || subjects?.mine.length === 0) && <BodyCopy>{t('subjects.list_page.no_data')}</BodyCopy>}
 				</StyledList>
+				<PrimaryLinkButton justifySelf="start">
+					<Link to={routes.subjects.subjectCreation}>{t('subjects.links.create')}</Link>
```

Comment:
Move the button somewhere more visible (or have another button for the same action that is more visible): I spent 5 minutes looking and then had to get the url from the `routes.js` file